projects/core/components/dialog/dialog.style.less

```diff
@@ -52,6 +56,14 @@
         border-radius: 0;
         border: none;
         padding: 48px ~'calc(50vw - 720px / 2)';
```

Comment:
How about we move this padding to `.dialog-host`? So that it is the one always responsible for padding.

Reply:
Also found a bug that between 600 and 720px this causes white paddings around content. This bug is present in current version, might as well fix it here while we're at it :) I think we can move `head` out of that new wrapper you introduced. And we can add border-radius to the wrapper only if it's the first child. Otherwise layout starts to feel a bit tricky.

Reply:
https://user-images.githubusercontent.com/4103756/106948734-5f4c7400-672c-11eb-8f2e-79f90647556c.mp4

Reply:
Should be fixed